test/pytest/test_bridge.py

```diff
@@ -658,7 +681,8 @@ async def open(
             **kwargs: JsonValue
     ) -> 'FsInfoClient':
         channel = await transport.check_open('fsinfo', path=str(path), attrs=attrs,
-                                             fnmatch=fnmatch, reply_keys=None, **kwargs)
+                                             fnmatch=fnmatch, reply_keys=None,
+                                             absent_keys=(), **kwargs)
```

Comment:
`absent_keys=()` is totally redundant here...

Reply:
I get a linter error then, likely due to **kwargs?